### DISCUSSION TITLE
"Delete Directory" displays on repositories not owned by myself

### DISCUSSION THREAD

**Discussion by andreaselia** [community/community#60381] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Please see the screenshot below. Also the icon isn't showing on the last option (if there is meant to be one).

<img width="311" alt="CleanShot 2023-07-08 at 09 02 33@2x" src="https://github.com/community/community/assets/5033092/2a1c6d61-87ab-45e1-9f95-8d866190c237">

**Comment by ronakjkothari** [comment]
You probably have permission to create, edit, and delete repositories

**Reply by andreaselia** [reply]
This isn't my repository though, it happens on all repositories.

**Comment by ronakjkothari** [comment]
I see the error. It prompts you to delete the directory, however, upon clicking the button, it will require you to create a fork of the project. It is just a way of proposing changes for other people. It won't actually delete the directory for their project, but rather create a fork of the project with the directory removed.

![image](https://github.com/community/community/assets/70562508/7bef0878-fa61-4086-85fb-07fdaae0f805)

Hope this helps!

**Reply by andreaselia** [reply]
But you wouldn't expect to see any red actionable text within someone else's repository, surely?

**Reply by ronakjkothari** [reply]
I agree, it shouldn't be red. Perhaps yellow if anything to show that there are errors with performing that action. Or maybe even grayed out to show that the action cannot be performed, and on hover, there would be a container prompting to fork the project.

**Reply by andreaselia** [reply]
I believe it should be hidden for those who don't have access to destructive actions, kinda like how only a repository owner can delete a repository/delete a file/etc.
